Clients/GUI/Pages/InstrumentGraphic.razor

```diff
@@ -0,0 +1,143 @@
+﻿@page "/instrument/{Figi}"
+@using ChartJs.Blazor.ChartJS.Common.Axes
+@using ChartJs.Blazor.ChartJS.Common.Axes.Ticks
+@using ChartJs.Blazor.ChartJS.Common.Enums
+@using ChartJs.Blazor.ChartJS.Common.Handlers
+@using ChartJs.Blazor.ChartJS.Common.Properties
+@using ChartJs.Blazor.ChartJS.Common.Time
+@using ChartJs.Blazor.ChartJS.LineChart
+@using ChartJs.Blazor.Charts
+@using GUI.Scripts
+@inject NavigationManager NavigationManager
+@inject ISessionStorageService Storage
+@inject IJSRuntime JsRuntime
+
+<ChartJsLineChart @ref="lineChartJs" Config="@lineConfig" Width="600" Height="300"/>
```

Comment:
Can you make the graph a bit brighter? (light colour blends with white background a bit)

Reply:
Color changed on blue